composer.json

```diff
@@ -43,6 +43,7 @@
         "symfony/http-foundation": "^4.3",
         "symfony/http-kernel": "^4.3",
         "symfony/options-resolver": "^4.3",
+        "symfony/polyfill-mbstring": "^1.10",
```

Comment:
do we need this?

Reply:
Yes and not. To reproduce functionality from twig/extensions it is (this package require it). To working code not becouse there is check for it.

https://github.com/sonata-project/SonataAdminBundle/pull/6145/files#diff-f89b944b94511771e65828324746216aR34-R57

WDYT? Should we keep it, add to suggest or ignore?

Reply:
The ìf` suggests this is not needed, or the if is not needed. IMO we could ignore it. @sonata-project/contributors wdyt?

Reply:
I agree with you @jordisala1991. Since there is a `if`, it's not needed.

Reply:
I remove it.